### PR TITLE
Adding arguments to Java and C#

### PR DESCRIPTION
**Pull request by diemol**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by titusfortner** [comment]
1. Should we add this to Edge & Firefox as well?
2. Should we add it to the actual documentation?

**Comment by diemol** [comment]
Yes to both.
Just trying to do one commit at a time.

File: examples/dotnet/SeleniumDocs/Browsers/FirefoxTest.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public void InstallUnsignedAddon()
         public void HeadlessOptions() 
         {
             var options = new FirefoxOptions();
-            options.AddArgument("-headless");
+            options.AddArgument("--headless");
```
Comment (titusfortner) [inline examples/dotnet/SeleniumDocs/Browsers/FirefoxTest.cs:75]: Firefox documentation has only one dash

Comment (diemol) [reply examples/dotnet/SeleniumDocs/Browsers/FirefoxTest.cs:75]: 👍

**Pull request merged**